src/audio/dai-zephyr.c

```diff
@@ -626,38 +580,41 @@ static int dai_capture_params(struct comp_dev *dev, uint32_t period_bytes,
 		config->dest_width = config->src_width;
 	}
 
-	comp_info(dev, "dai_capture_params() src_dev = %d stream_id = %d src_width = %d dest_width = %d",
-		  config->src_dev, dd->stream_id,
-		  config->src_width, config->dest_width);
+	comp_dbg(dev, "dai_set_sg_config():");
+	comp_dbg(dev, "src_dev = %d dest_dev = %d stream_id = %d src_width = %d dest_width = %d",
+		 config->src_dev, config->dest_dev, dd->stream_id, config->src_width,
+		 config->dest_width);
 
 	if (!config->elem_array.elems) {
-		fifo = dai_get_fifo(dd->dai, dev->direction,
-				    dd->stream_id);
+		fifo = dai_get_fifo(dd->dai, dev->direction, dd->stream_id);
 
-		comp_dbg(dev, "dai_capture_params() fifo 0x%x", fifo);
+		comp_dbg(dev, "dai_set_sg_config(): fifo 0x%x", fifo);
 
-		err = dma_get_attribute(dd->dma->z_dev, DMA_ATTR_MAX_BLOCK_COUNT,
-					&max_block_count);
+		err = dma_get_attribute(dd->dma->z_dev, DMA_ATTR_MAX_BLOCK_COUNT, &max_block_count);
 		if (err < 0) {
-			comp_err(dev, "dai_capture_params(): could not get dma attr max block count, err = %d", err);
+			comp_err(dev, "dai_set_sg_config(): can't get max block count, err = %d",
+				 err);
 			goto out;
 		}
 
 		if (!max_block_count) {
-			comp_err(dev, "dai_capture_params(): invalid max-block-count of zero");
+			comp_err(dev, "dai_set_sg_config(): invalid max-block-count of zero");
 			goto out;
 		}
 
 		if (max_block_count < period_count) {
-			comp_dbg(dev, "dai_capture_params(): block count = %d not supported by DMA", period_count);
+			comp_dbg(dev, "dai_set_sg_config(): unsupported period count %d",
+				 period_count);
 			buf_size = period_count * period_bytes;
 			do {
 				if (IS_ALIGNED(buf_size, max_block_count)) {
 					period_count = max_block_count;
 					period_bytes = buf_size / period_count;
 					break;
 				} else {
-					comp_warn(dev, "dai_capture_params(): alignment error for buffer = %d, block count = %d", buf_size, max_block_count);
+					comp_warn(dev, "dai_set_sg_config() alignment error for");
+					comp_warn(dev, "buf_size = %d", buf_size);
+					comp_warn(dev, "block count = %d", max_block_count);
```

Comment:
I think I'd leave these alone - if you want to make lines shorter, which would be good here, just break it at some of the commas, but one warning should be less intrusive than 3. Same in line 672 / 629 below